ui/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx

```diff
@@ -149,20 +147,21 @@ class ResultsExport extends React.Component<Props, State> {
           transformerId: uriEncodedTransformerId,
         },
       })
-    } else if (this.props.results.length > 1) {
+    } else {
+      let attributes = new Array()
```

Comment:
❓ Why can't we just do `const attributes = this.props.results.map(result => result.attributes)`?

Reply:
You're right, I think I could do something like

`const attributes = Array.from(new Set(this.props.results.map(result => result.attributes)))` which would be more concise. The Set part is just so that I can get only the unique values

Reply:
If you only want unique values you can use the `distinct()` stream operator like `this.props.results.map(result => result.attributes).distinct()`

Reply:
this is exactly what i was looking for - thanks